docs/DeveloperGuide.adoc

```diff
@@ -1775,34 +1775,76 @@ These instructions only provide a starting point for testers to work on; testers
 
 .. Download the jar file and copy into an empty folder
 .. Double-click the jar file +
-   Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   Expected: Shows the GUI. You should be greeted with a prompt to login or register. The window size may not be optimum.
 
 . Saving window preferences
 
 .. Resize the window to an optimum size. Move the window to a different location. Close the window.
 .. Re-launch the app by double-clicking the jar file. +
    Expected: The most recent window size and location is retained.
 
-_{ more test cases ... }_
+=== Registering a Student Account
```

Comment:
Might be good to include an example of creating a student without a minor. Not necessary but it shows that it is possible.